### ISSUE TITLE
Nanominer shows two GPUs. Only one installed on the system.

### ISSUE THREAD

**Issue by getsec** [nanopool/nanominer#213]
GPU: AMD RX 6700 XT.
MEM: 32
CPU: i5 9600k @ 4400Mhz

**Comment by Raptor2213** [comment]
Your machine likely has onboard graphics that it's detecting.

**Comment by snok87** [comment]
> Your machine likely has onboard graphics that it's detecting.

Nope, sometimes it shows twice same card on same PCI port. I just try to open, close, open close and if don't work I have to edit my nanominer config to select the correct GPU devices. It's a pain.
